From e6316792f6b9bc4dad404ac4a341328c1c86caf3 Mon Sep 17 00:00:00 2001
From: Josua Mayer <josua@solid-run.com>
Date: Mon, 26 May 2025 15:46:55 +0300
Subject: [PATCH 3/4] usb: misc: onboard-hub: add of_match for cypress hx3
 alternate pids

Cypress HX3 can appear as various product IDs, add known IDs so that
device-tree node for those ids can be matched.

Signed-off-by: Josua Mayer <josua@solid-run.com>
---
 drivers/usb/misc/onboard_usb_hub.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/usb/misc/onboard_usb_hub.h b/drivers/usb/misc/onboard_usb_hub.h
index 8af34e6d1aff..5148ce2658b6 100644
--- a/drivers/usb/misc/onboard_usb_hub.h
+++ b/drivers/usb/misc/onboard_usb_hub.h
@@ -59,6 +59,8 @@ static const struct of_device_id onboard_hub_match[] = {
 	{ .compatible = "usb424,5744", .data = &microchip_usb5744_data, },
 	{ .compatible = "usb451,8140", .data = &ti_tusb8041_data, },
 	{ .compatible = "usb451,8142", .data = &ti_tusb8041_data, },
+	{ .compatible = "usb4b4,6500", .data = &cypress_hx3_data, },
+	{ .compatible = "usb4b4,6502", .data = &cypress_hx3_data, },
 	{ .compatible = "usb4b4,6504", .data = &cypress_hx3_data, },
 	{ .compatible = "usb4b4,6506", .data = &cypress_hx3_data, },
 	{ .compatible = "usb5e3,608", .data = &genesys_gl850g_data, },
-- 
2.51.0

